module7_selenium/src/test/java/BringItOnTestMain.java

```diff
@@ -0,0 +1,74 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import page.PastebinResultPage;
+import page.PastebinStartPage;
+
+import java.util.concurrent.TimeUnit;
+
+public class BringItOnTestMain {
```

Comment:
You can create base test class with @BeforeClass, @BeforeMethod, @AfterMethod annotated methods and extend your test classes from this base class.